core/src/main/scala/cats/data/Validated.scala

```diff
@@ -228,10 +227,25 @@ private[data] sealed abstract class ValidatedInstances extends ValidatedInstance
     def show(f: Validated[A,B]): String = f.show
   }
 
-  implicit def validatedBifunctor: Bifunctor[Validated] =
-    new Bifunctor[Validated] {
-      override def bimap[A, B, C, D](fab: Validated[A, B])(f: A => C, g: B => D): Validated[C, D] = fab.bimap(f, g)
-      override def leftMap[A, B, C](fab: Validated[A, B])(f: A => C): Validated[C, B] = fab.leftMap(f)
+  implicit def validatedBifunctor: Bitraverse[Validated] =
```

Comment:
A couple other notes:

I know this was already a `def`, but since I noticed it in the diff: could we switch it to a `val` to avoid repeated allocations? Or does it hit the #965 strangeness?

Are we keeping the name `validatedBifunctor` for compatibility reasons? If so, we should probably leave a comment that that's what we are doing. However, I think we can feel free to change this to `validatedBitraverse` since we won't really start to focus on compatibility until a 1.0 release (hopefully coming soon!).